### DISCUSSION TITLE
How to migrate to localhost

### DISCUSSION THREAD

**Discussion by deepslam** [prisma/prisma#11451] [Q&A]
Hi everyone!
Could you please help me?
I'm setting up everything from the beginning on my new laptop and I want to transfer my Prisma database from my website to my localhost.
It transferred ok, I created and upload a dump.
But when I run

> npx prisma migrate dev

It offers me to remove all the data and recreate a database.
It seems not right and how can I avoid it?
Thanks!

**Comment by Skryvvara** [comment]
I might be wrong but if you imported a dump of your entire db sucessfully you dont need to run prisma migrate anymore. Prisma migrate creates the tables and relations for you (what you did with importing the dump). To generate the code to write queries run npx prisma generate

**Reply by vladi-stevanovic** [reply]
👋 Hello @deepslam I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! :raised_hands:

**Comment by janpio** [comment]
When `migrate dev` runs the first time, it expects to migrate from an empty database to what the Prisma schema defines. If that is not the case, it will suggest to reset.

The correct workflow to avoid this, is to "baseline" the database first before running `migrate dev`: https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/baselining This creates an initial migration that reflects the inital state of the database.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
